SharkCage/CageManager/CageManager.cpp

```diff
@@ -89,6 +119,15 @@ void CageManager::StartCage(SECURITY_ATTRIBUTES security_attributes, const CageD
 
 	::RpcStringFree(&uuid_str);
 	
+	HANDLE tokenHandle = 0;
+	//TODO: close this handle
+	if (!tokenLib::constructUserTokenWithGroup(static_cast<LPWSTR>(const_cast<wchar_t*>((group_name.c_str()))), tokenHandle)) {
+		std::cout << "Cannot create required token" << std::endl;
+		getchar();
```

Comment:
Why you're calling `getchar` at this point?

Reply:
Leftover from debug, to keep console from  closing... 😅